Submitted By: Jim Gifford (jim at linuxfromscratch dot org)
Date: 2004-06-24
Initial Package Version: 2.6.7
Origin: Jim Gifford
Upstream Status: Sent
Description: Adds Missing Header to arpd.c
 
--- iproute2-2.6.7/misc/arpd.c.orig	2004-06-24 18:39:06.370510664 +0000
+++ iproute2-2.6.7/misc/arpd.c	2004-06-24 18:39:18.197399462 +0000
@@ -27,6 +27,7 @@
 #include <time.h>
 #include <signal.h>
 #include <linux/if.h>
+#include <linux/if_ether.h>
 #include <linux/if_arp.h>
 #include <netinet/in.h>
 #include <arpa/inet.h>
